### PR TITLE
Test api-functional + integration tests speed

### PR DESCRIPTION
**Pull request by skovalenk**
### Related Pull Requests

https://github.com/magento/partners-magento2-infrastructure/pull/38

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by skovalenk** [comment]
@magento run Integration Tests

**Comment by skovalenk** [comment]
@magento run WebAPI Tests

**Comment by skovalenk** [comment]
@magento run WebAPI Tests

File: lib/internal/Magento/Framework/MessageQueue/CallbackInvoker.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ public function invoke(
         $sleep = null
     ) {
         $this->poisonPillVersion = $this->poisonPillRead->getLatestVersion();
-        $sleep = (int) $sleep ?: 1;
+        $sleep = 0; //(int) $sleep ?: 1;
```
Comment (mslabko) [inline lib/internal/Magento/Framework/MessageQueue/CallbackInvoker.php]: can we use usleep?

File: lib/internal/Magento/Framework/MessageQueue/CallbackInvoker.php [hunk 2]
```diff
@@ -103,6 +103,6 @@ public function invoke(
      */
     private function isWaitingNextMessage(): bool
     {
-        return $this->deploymentConfig->get('queue/consumers_wait_for_messages', 1) === 1;
+        return false; //$this->deploymentConfig->get('queue/consumers_wait_for_messages', 1) === 1;
```
Comment (mslabko) [inline lib/internal/Magento/Framework/MessageQueue/CallbackInvoker.php]: it will break part of the functionality.

**Pull request closed without merging**